### PR TITLE
Update CHANGES for release 12.2.0

### PR DESCRIPTION
**Pull request by jeremy-lunarg**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -3,6 +3,19 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/).
 
+## 12.2.0 2023-05-17
+
+### Other changes
+* Add support GLSL_EXT_shader_tile_image
+* Add support for GL_EXT_ray_tracing_position_fetch
+* Use custom callbacks if they are available
```
Comment (arcady-lunarg) [inline CHANGES.md:11]: Perhaps reword this to
"Support custom include callbacks via the C API"?

Comment (jeremy-lunarg) [reply CHANGES.md]: Good call. Thanks!

**Review by arcady-lunarg — APPROVED** [review]
LGTM
